src/libsync/discoveryphase.cpp

```diff
@@ -583,14 +583,20 @@ void DiscoverySingleDirectoryJob::lsJobFinishedWithoutErrorSlot()
 
 void DiscoverySingleDirectoryJob::lsJobFinishedWithErrorSlot(QNetworkReply *r)
 {
-    QString contentType = r->header(QNetworkRequest::ContentTypeHeader).toString();
-    int httpCode = r->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
+    const QString contentType = r->header(QNetworkRequest::ContentTypeHeader).toString();
+    const bool invalidContentType = !contentType.contains("application/xml; charset=utf-8") &&
+                                    !contentType.contains("application/xml; charset=\"utf-8\"") &&
```

Comment:
what can produce `"utf-8"` ?
looks like a bogus value to me (see https://www.rfc-editor.org/rfc/rfc7303.html#page-15)

Reply:
It's a valid format: https://httpwg.org/specs/rfc7231.html#data.type

Reply:
OK, thanks for finding the reference as I could not find it